src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
+    
+    If Not changeFileExtension(ThisWorkbook.Path, ThisWorkbook.Name) Then
```

Comment:
"ThisWorkbook" refers to the Add-in itself. Perhaps using "ActiveWorkbook" instead would be more useful for the user?